SpatialGDK/Extras/core-sdk.version

```diff
@@ -1,3 +1,3 @@
-15.3.1
+15.4.0-b942-aa3ea09-WORKER-SNAPSHOT

```

Comment:
Do we know when the next release is coming?

Reply:
We'll be running custom versions from now on